scripts/webservice-image-digest.py

```diff
@@ -46,20 +54,26 @@ def get_commit_from_github(tag_or_branch):
   print("No commit for that tag or branch found!")
   exit(1)
 
-def get_digest_from_s3(tag, commit):
+def get_digest_from_s3(directory):
   # downloads the image-digest.txt from a directory in S3
   base_url = "https://gui.dockstore.org"
-  response = requests.get("{}/{}-{}/image-digest.txt".format(base_url, tag, commit[0:7]))
+  response = requests.get("{}/{}/image-digest.txt".format(base_url, directory))
   if (response.status_code != 200):
-    print("Expected a file at {}".format("{}/{}-{}/image-digest.txt".format(base_url, tag, commit[0:7])))
+    print("Expected a file at {}".format("{}/{}/image-digest.txt".format(base_url, directory)))
     print("The image-digest.txt was not found in S3, did the build succeed?")
     exit(1)
   # There is a newline at the end of the file we rstrip
   return response.text.rstrip()
 
 if __name__ == "__main__":
   # slashes are replaced with _ in docker image tags
-  commit = get_commit_from_github(args.tag)
-  circle_digest = get_digest_from_s3(args.tag, commit)
+  # check to see if input includes a dash followed by 7 chars
+  parsed = args.tag.split('-')
+  if len(parsed) == 2 and len(parsed[1]) == 7:
```

Comment:
This is a super-paranoid suggestion, but maybe we also check if the shorthash is hex.